x-pack/test/api_integration/apis/uptime/rest/helpers/make_checks.ts

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import uuid from 'uuid';
```

Comment:
I'm confused about this also, this file seems to exist in both `rest` and `graphql` dirs now.

Reply:
yes i was just testing , now i have removed and have handled the use case in same file.
Thanks you for catching this.